### PR TITLE
drop 1.28 jobs as it is eol

### PR DESCRIPTION
**Pull request by cpanato**
Fixes: https://github.com/kubernetes/test-infra/issues/33883

/assign @saschagrunert @dims @xmudrii @Verolop 
cc @kubernetes/release-managers

### CODE REVIEW

File: releng/test_config.yaml [hunk 1]
```diff
@@ -161,44 +161,44 @@ jobs:
...
-    interval: 24h
-    args:
-    - --env=KUBE_PROXY_DAEMONSET=true
-    - --env=ENABLE_POD_PRIORITY=true
-    # Panic if anything mutates a shared informer cache
-    - --env=ENABLE_CACHE_MUTATION_DETECTOR=true
-    - --runtime-config=api/beta=true
-    sigOwners: [sig-cloud-provider-gcp]
-    releaseBlocking: true
+  # ci-kubernetes-e2e-gce-cos-k8sstable4-ingress:
```
Comment (dims) [inline releng/test_config.yaml:164]: commenting these on purpose i think (until we need it again). right?

**Comment by dims** [comment]
/skip
/approve

one question inline.. LGTM as well.

**Comment by dims** [comment]
will `/lgtm` once the failing CI job is fixed. looks like a dashboard needs to be cleaned up

```
could not apply prowjob annotations: couldn't find dashboard "sig-release-1.28-informing" for job "ci-kubernetes-e2e-capz-windows-1-28"
FAIL	k8s.io/test-infra/config/tests/testgrids	1.959s
```

File: releng/test_config.yaml (hunk 1 above)
Comment (cpanato) [reply releng/test_config.yaml:164]: yes correct

**Pull request merged**